lib/orcid_client/version.rb

```diff
@@ -1,3 +1,3 @@
 module OrcidClient
-  VERSION = "0.11.0"
+  VERSION = "0.11.1"
```

Comment:
Minor version please, not micro. 
I'd have said a Major but we're not really using semantic here very well.
This gem is basically only used by us, so it doesn't matter too much but I think 0.12 makes more sense, in case for any reason we have to patch in a previous version.

Reply:
@richardhallett thank you. i've made the change.